src/components/NavBar/Navbar.jsx

```diff
@@ -41,10 +42,7 @@ function Navbar() {
             </li>
           </ul>
           <div className="d-flex ms-3">
-            <Link
-              className="btn btn-outline-light custom-btn me-2"
-              to="/signup"
-            >
+            <Link className="btn btn-outline-light custom-btn me-2" to="/login">
```

Comment:
This is the SignUp button and does not need to take us to the login page. If they are going to be the same page, then let's get rid of the second link instead.